Makefile

```diff
@@ -173,6 +173,9 @@ install: all
 	install -m 0755 dracut-initramfs-restore.sh $(DESTDIR)$(pkglibdir)/dracut-initramfs-restore
 	rm -rf $(DESTDIR)$(pkglibdir)/modules.d/80test*
 	cp -arx modules.d dracut.conf.d $(DESTDIR)$(pkglibdir)
```

Comment:
Copying the complete dracut.conf.d directly beforehand does not make sense to me (especially after this merge request). But feel free to merge this one and address my remark in a later one.